Gruntfile.js

```diff
@@ -162,6 +162,13 @@ module.exports = function(grunt) {
           base: '.',
           port: 9998
         }
+      },
+      https: {
+        options: {
+          base: '.',
+          port: 9998,
+          protocol: 'https'
```

Comment:
this is necessary because chrome has a few weird security rules:

1. crypto apis are not available on http
2. service workers are not available on http or file://